config.libsonnet

```diff
@@ -1,6 +1,16 @@
 {
   _config+:: {
+    // Selectors are inserted between {} in Prometheus queries.
     glusterExporterSelector: 'job="glusterd2-client"',
     volumeUtilizationAlertTime: '5m',
+
+    // Grafana dashboard IDs are necessary for stable links for dashboards
+    grafanaDashboardIDs: {
+      'k8s-storage-resources-glusterfs-pv.json': 'XnbvYbcXkob7GLqcDPLTj1ZL4MRX87tOh8xdr567',
```

Comment:
Does this ID match the ID attached by `default.libsonnet` ?

Reply:
No it does not.